feat(web): custom reset password link (#2854)
authelia/authelia: docs/configuration/authentication/index.md

```yaml
authentication_backend:
  disable_reset_password: false

>   enable_external_reset_password: false
>   external_reset_password_url: ""

Reviewer: ```suggestion
  password_reset:
    custom_url: ""
```

This setting controls if users can reset their password from the web frontend or not.

> ### enable_external_reset_password
> <div markdown="1">
> type: boolean
> {: .label .label-config .label-purple } 
> default: false
> {: .label .label-config .label-blue }
> required: no
> {: .label .label-config .label-green }
> </div>
> 
> This setting controls if users can reset their password from an external reset password portal.
> 
> ### external_reset_password_url
> <div markdown="1">
> type: string
> {: .label .label-config .label-purple } 
> default: ""
> {: .label .label-config .label-blue }
> required: no (yes if enable_external_reset_password is true)
> {: .label .label-config .label-green }
> </div>
> 
> The custom reset password URL, which redirects users to the custom reset password portal. 

Reviewer: ```suggestion
### password_reset

Password reset options. 

#### custom_url
<div markdown="1">
type: string
{: .label .label-config .label-purple } 
default: ""
{: .label .label-config .label-blue }
required: no
{: .label .label-config .label-green }
</div>

The custom reset password URL, which redirects users to the custom reset password portal. 
```
